core/src/main/scala/kafka/Kafka.scala

```diff
@@ -86,6 +86,10 @@ object Kafka extends Logging {
   }
 
   def main(args: Array[String]): Unit = {
+    process(args)
+  }
+
+  def process(args: Array[String]): Unit = {
```

Comment:
Why is `process` needed? You can invoke `main` from `KafkaNativeWrapper`.

Reply:
Done